.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
### Your MIT.md file :spiral_notepad:
![mitfile (copy)](https://user-images.githubusercontent.com/93039359/186214250-99a92579-8752-4208-9c5d-1084ce85421d.png)


- [ ] Please add your name, current year, app type in your MIT.md file. This is the license file of your project so please take it seriously :thinking: